draft/draft-ietf-rtgwg-qos-model.xml

```diff
@@ -366,187 +369,201 @@
       </section>
 
-      <section title="ietf-diffserv">
+      <section title="Diffserv Module">
+	<t>Diffserv module augments Traffic Policy module.</t>
+
+	<figure>
+	  <name>ietf-diffserv tree diagram</name>
+          <artwork><![CDATA[
+	  INSERT_TEXT_FROM_FILE(../bin/ietf-diffserv@YYYY-MM-DD-sub-tree.txt)
+	  ]]></artwork>
+	</figure>
+
         <figure>
 	  <name>ietf-diffserv module</name>
           <artwork><![CDATA[
-<CODE BEGINS>file "ietf-diffserv@YYYY-MM-DD.yang"
-INSERT_TEXT_FROM_FILE(../bin/ietf-diffserv@YYYY-MM-DD.yang)
-<CODE ENDS>
-    ]]></artwork>
+	  <CODE BEGINS>file "ietf-diffserv@YYYY-MM-DD.yang"
+	  INSERT_TEXT_FROM_FILE(../bin/ietf-diffserv@YYYY-MM-DD.yang)
+	  <CODE ENDS>
+	  ]]></artwork>
         </figure>
       </section>
-      <section title="ietf-queue-policy">
+
+      <section title="Qos Operational Module">
+	<t>QoS Operational module contains all operational
+	statistics. It contains statistics related to classifier,
+	metering, queueing, and named.</t>
+
+	<t>Classifier statistics consist of list of classifier entries
+	identified by a classifier entry name.  Classifier counters
+	include matched packets and bytes, and average rate of traffic
+	matching a particular classifier.</t>
+
+	<t>Metering statistics consist of meters identified by an
+	identifier.  Metering counters include conform, exceed,
+	violate, drop packets, and bytes.</t>
+
+	<t>Queuing counters include instantaneous, peak, average queue
+	length, as well as output conform, exceed, tail drop packets
+	and bytes.</t>
+
+	<t>Named statistics is defined as statistics which is tagged
+	by a name.  This could be aggregated or non-aggregated.
+	Aggregated named statistics is defined as counters which are
+	aggregated across classifier entries in a policy applied to an
+	interface in a particular direction.  Non-aggregated named
+	statistics are counters of classifier, metering or queuing
+	which have the same tag name but maintained separately.</t>
+
+	<t>A clear action is provided to clear all the statistics.</t>
+
+	<figure>
+	  <name>ietf-qos-oper tree diagram</name>
+          <artwork><![CDATA[
+	  INSERT_TEXT_FROM_FILE(../bin/ietf-qos-oper@YYYY-MM-DD-sub-tree.txt)
+	  ]]></artwork>
+	</figure>
         <figure>
-	  <name>ietf-queue-policy module</name>
+	  <name>ietf-qos-oper module</name>
           <artwork><![CDATA[
-<CODE BEGINS>file "ietf-queue-policy@YYYY-MM-DD.yang"
-INSERT_TEXT_FROM_FILE(../bin/ietf-queue-policy@YYYY-MM-DD.yang)
-<CODE ENDS>
-    ]]></artwork>
+	  <CODE BEGINS>file "ietf-qos-oper@YYYY-MM-DD.yang"
+	  INSERT_TEXT_FROM_FILE(../bin/ietf-qos-oper@YYYY-MM-DD.yang)
+	  <CODE ENDS>
+	  ]]></artwork>
         </figure>
       </section>
-      <section title="ietf-scheduler-policy">
+
+      <section title="Scheduler Policy Module">
+	<t> Scheduling Policy module defines a set of scheduling
+	parameters and associates a Queue Policy with it(Note, why is the queue-policy a string rather than a reference).</t>
```

Comment:
yes it can be, we have avoided references.. since then it would be asked in all places and we have trouble between inline-classifier and referred classifier..